build/driver.go

```diff
@@ -179,6 +194,7 @@ func (r *nodeResolver) resolve(ctx context.Context, ps []specs.Platform, pw prog
 			resolver:    r,
 			driverIndex: 0,
 		})
+		nodeIdxs = append(nodeIdxs, 0)
```

Comment:
This seemed to be another issue. Atm. because first boot was called with empty index array this accidentally saved us from one more extra uncached call (the real first boot happened when `opts()` was first called). Now there is one more call to `boot()` but because all calls are cached the number of calls does not matter anymore.